sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -631,80 +665,55 @@ def query_items_change_feed( # pylint: disable=unused-argument
+            partition_key = kwargs["partition_key"]
+            change_feed_state_context["partitionKey"] = self._set_partition_key(cast(PartitionKeyType, partition_key))
+            change_feed_state_context["partitionKeyFeedRange"] = self._get_epk_range_for_partition_key(partition_key)
+        if "feed_range" in kwargs:
+            change_feed_state_context["feedRange"] = kwargs['feed_range']
+        if "continuation" in feed_options:
+            change_feed_state_context["continuation"] = feed_options.pop("continuation")
 
-            if kwargs.get("start_time") is not None:
-                raise ValueError("is_start_from_beginning and start_time are exclusive, please only set one of them")
-
-            is_start_from_beginning = kwargs.pop('is_start_from_beginning')
-            if is_start_from_beginning is True:
-                change_feed_state_context["startTime"] = "Beginning"
-
-        # parse start_time
-        if kwargs.get("start_time") is not None:
-            start_time = kwargs.pop('start_time')
-            if not isinstance(start_time, (datetime, str)):
-                raise TypeError(
-                    "'start_time' must be either a datetime object, or either the values 'Now' or 'Beginning'.")
-            change_feed_state_context["startTime"] = start_time
-
-        # parse continuation token
-        if feed_options.get("continuation") is not None:
-            change_feed_state_context["continuation"] = feed_options.pop('continuation')
-
-        if kwargs.get("max_item_count") is not None:
-            feed_options["maxItemCount"] = kwargs.pop('max_item_count')
-
-        if kwargs.get("partition_key") is not None:
-            change_feed_state_context["partitionKey"] =\
-                self._set_partition_key(cast(PartitionKeyType, kwargs.get("partition_key")))
-            change_feed_state_context["partitionKeyFeedRange"] = \
-                self._get_epk_range_for_partition_key(kwargs.pop('partition_key'))
-
-        if kwargs.get("feed_range") is not None:
-            change_feed_state_context["feedRange"] = kwargs.pop('feed_range')
-
-        feed_options["containerProperties"] = self._get_properties()
         feed_options["changeFeedStateContext"] = change_feed_state_context
+        feed_options["containerProperties"] = self._get_properties()
 
-        response_hook = kwargs.pop('response_hook', None)
+        response_hook = kwargs.pop("response_hook", None)
         if hasattr(response_hook, "clear"):
             response_hook.clear()
 
         if self.container_link in self.__get_client_container_caches():
             feed_options["containerRID"] = self.__get_client_container_caches()[self.container_link]["_rid"]
 
         result = self.client_connection.QueryItemsChangeFeed(
-            self.container_link, options=feed_options, response_hook=response_hook, **kwargs
-        )
+            self.container_link, options=feed_options, response_hook=response_hook)
```

Comment:
why remove **kwargs

Reply:
Mb, I thought it was not used later in the code, but I found one place we used it. I put it back again